src/seedu/addressbook/parser/Parser.java

```diff
@@ -11,17 +11,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.addressbook.commands.AddCommand;
-import seedu.addressbook.commands.ClearCommand;
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```

Comment:
> 
> 
> Great to see you have submitted your changes via your own branch and done the documentation. 👍
> 
> You might want to try to write the test first before implementing the functionality to adopt TDD workflow.
> 
> You have demonstrated the progress of this exercise through the commits you have done. Maybe, as you can more comfortable, your commits can be done in batch, by committing the test, the implementation and the relevant documentation together. instead of committing changes in each file separately. This way, it is easier to navigate through commits in events you need to revert to certain state of the code, and all its related dependencies (such as documentation, tests) are in sync. Otherwise, for example, you can revert the commit for the code change but not the documentation (if the documentation is written prior to the code).
> 
> Please close this PR after reading comments. Thanks.

Thanks for your timely feedback. I will take note of them and apply it to subsequent tasks and the project.